### PR TITLE
feat: update compressed-nfts.md to use umi and bubblegum plugin

### PR DESCRIPTION
**Pull request by jim4067**
### Summary of Changes
update compressed nfts lesson to use umi and mpl-bubblegum package

Fixes #

### CODE REVIEW

**Review by mikemaccana — CHANGES_REQUESTED** [review]
Great work! Lots of small changes (and a few TODOs) but you're 90% of the way there and will likely win this one.

File: content/courses/state-compression/compressed-nfts.md [hunk 1]
```diff
@@ -326,215 +320,152 @@ is 32 bytes it’s possible to max out transaction sizes very quickly.
...
-address as a seed.
+// load keypair from local file system
+// See https://github.com/solana-developers/helpers?tab=readme-ov-file#get-a-keypair-from-a-keypair-file
+const localKeypair = await getKeypairFromFile();
+
+// convert to Umi compatible keypair
+const umiKeypair = umi.eddsa.createKeypairFromSecretKey(localKeypair.secretKey);
+
+// load the MPL Bubblegum program, dasApi plugin and assign a signer to our umi instance
+umi.use(keypairIdentity(umiKeypair)).use(mplBubblegum()).use(dasApi());
```
[the remark this answers is not in the document]
Comment (jim4067) [reply content/courses/state-compression/compressed-nfts.md:385]: prettier formats this to one line in the md

File: content/courses/state-compression/compressed-nfts.md [hunk 2]
```diff
@@ -820,540 +634,391 @@ from a Merkle tree.
...
+```
+
+I am going to be
+[recycling a Collection NFT](https://explorer.solana.com/address/D2zi1QQmtZR5fk7wpA1Fmf6hTY2xy8xVMyNgfq6LsKy1?cluster=devnet)
+I already created in the NFTs with Metaplex lesson, but if you'd like to create
+a new collection for this lesson, check out the code
+[on this repo](https://github.com/solana-developers/professional-education/blob/main/labs/metaplex-umi/create-collection.ts)
+
+<!-- todo(should there be a section for creating a collection with MPL metadata) -->
```
[the remark this answers is not in the document]
Comment (jim4067) [reply content/courses/state-compression/compressed-nfts.md:789]: I linked the section for creating a collection NFT in the nfts-with-metaplex lesson

File: content/courses/state-compression/compressed-nfts.md [hunk 3]
```diff
@@ -1362,287 +1027,99 @@ owner, creator, etc., and more. Be sure to look through the
...
 
-<Callout type="note">If you plan to mint a large amount of NFTs you might want
-to try and batch these instructions for fewer total transactions.</Callout>
-
-If you need more time with this lab, feel free to go through it again and/or
-take a look at the solution code on the `solution` branch of the
-[lab repo](https://github.com/Unboxed-Software/solana-cnft-demo/tree/solution).
+Inspect the updated NFT on Solana Explorer! Just like previously, if you have
+any issues, you should fix them yourself, but if needed the
+[solution code](<!-- todo(Jimii) -->) is available.
```
[the remark this answers is not in the document]
Comment (jim4067) [reply content/courses/state-compression/compressed-nfts.md:1127]: Here is the demo repo -> https://github.com/jim4067/cnft-demo

File: content/courses/state-compression/compressed-nfts.md [hunk 4]
```diff
@@ -820,540 +634,391 @@ from a Merkle tree.
...
-  const treeKeypair = Keypair.generate();
-
-  const allocTreeIx = await createAllocTreeIx(
-    connection,
-    treeKeypair.publicKey,
-    payer.publicKey,
-    maxDepthSizePair,
-    canopyDepth,
-  );
-}
```
[the remark this answers is not in the document]
Comment (jim4067) [reply content/courses/state-compression/compressed-nfts.md:896]: sounds awesome. Let me look a little bit more into it

File: content/courses/state-compression/compressed-nfts.md [hunk 5]
```diff
@@ -1117,8 +1122,8 @@ larger values and as long as you have enough Devnet SOL, this script will let
 you mint up to 10k cNFTs for a small fraction of what it would cost to mint 10k
 traditional NFTs.
 
-Inspect the updated NFT on Solana Explorer! Just like previously, if you have
-any issues, you should fix them yourself, but if needed the
+Inspect the cNFT on Solana Explorer! Just like previously, if you have any
+issues, you should fix them yourself, but if needed the
 [solution code](<!-- todo(Jimii) -->) is available.
```
Comment (mikemaccana) [inline content/courses/state-compression/compressed-nfts.md:1127]:
```suggestion
[solution code](https://github.com/solana-foundation/compressed-nfts) is available.
```

**Review by mikemaccana — APPROVED** [review]

**Pull request merged**